Makefile

```diff
@@ -494,9 +523,29 @@ $(INCLUDE_PATH)/oidc-agent/oidc_error.h: $(SRCDIR)/utils/oidc_error.h $(INCLUDE_
 $(LIBDEV_PATH)/liboidc-agent.a: $(APILIB)/liboidc-agent.a $(LIBDEV_PATH)
 	@install $< $@
 
-$(INCLUDE_PATH)/oidc-agent/export_symbols.h: $(SRCDIR)/$(CLIENT)/export_symbols.h $(INCLUDE_PATH)/oidc-agent
+$(INCLUDE_PATH)/oidc-agent/api/export_symbols.h: $(SRCDIR)/api/export_symbols.h $(INCLUDE_PATH)/oidc-agent/api
```

Comment:
Why is `export_symbols.h` in a own folder `api` and others like `api.h` are not.
I would put everything directly under `oidc-agent` with no `api` folder and then including the `export_symbols.h` with
```c
#include "export_symbols.h"
```

Reply:
The reason is that both `oidc-add/api.h` and `oidc-token/api.h` include `api/export_symbols.h`. When we export both `api.h` files to include folder, they still contain the reference to `api/export_symbols.h`. To overcome this drawback, I decided to create a subfolder in the include folder. What would be the better way to proceed?

Reply:
I'll check